### PR TITLE
Add some ref packages from 5.0.0

### PR DESCRIPTION
**Pull request by dagood**

### CODE REVIEW

**Review by crummel — APPROVED** [review]

File: src/referencePackages/src/system.runtime.compilerservices.unsafe/5.0.0/ref/netstandard2.1/System.Runtime.CompilerServices.Unsafe.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        public static ref T AddByteOffset<T>(ref T source, System.IntPtr byteOffset) { throw null; }
+        public unsafe static void* Add<T>(void* source, int elementOffset) { throw null; }
+        public static ref T Add<T>(ref T source, int elementOffset) { throw null; }
+        public static ref T Add<T>(ref T source, System.IntPtr elementOffset) { throw null; }
+        public static bool AreSame<T>([System.Diagnostics.CodeAnalysis.AllowNullAttribute]ref T left, [System.Diagnostics.CodeAnalysis.AllowNullAttribute]ref T right) { throw null; }
+        public unsafe static void* AsPointer<T>(ref T value) { throw null; }
+        public unsafe static ref T AsRef<T>(void* source) { throw null; }
+        public static ref T AsRef<T>(in T source) { throw null; }
+        // Manual fixup on next line. Attribute was in wrong place and not marked "return:". https://github.com/dotnet/arcade/issues/6718
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("o")] public static T As<T>(object o) where T : class { throw null; }
```
Comment (dagood) [inline src/referencePackages/src/system.runtime.compilerservices.unsafe/5.0.0/ref/netstandard2.1/System.Runtime.CompilerServices.Unsafe.cs:45]: https://github.com/dotnet/arcade/issues/6718

**Comment by dagood** [comment]
Filed https://github.com/dotnet/source-build/issues/1956 for missing `LangVersion`.

File: src/referencePackages/src/system.runtime.compilerservices.unsafe/5.0.0/lib/netcoreapp2.0/System.Runtime.CompilerServices.Unsafe.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        public unsafe static T ReadUnaligned<T>(void* source) { throw null; }
+        public unsafe static T Read<T>(void* source) { throw null; }
+        public static int SizeOf<T>() { throw null; }
+        public static void SkipInit<T>(out T value) { throw null; }
+        public static ref T SubtractByteOffset<T>(ref T source, System.IntPtr byteOffset) { throw null; }
+        public unsafe static void* Subtract<T>(void* source, int elementOffset) { throw null; }
+        public static ref T Subtract<T>(ref T source, int elementOffset) { throw null; }
+        public static ref T Subtract<T>(ref T source, System.IntPtr elementOffset) { throw null; }
+        // Manual fixup on next line. Was "where T : ValueType", which is invalid. https://github.com/dotnet/arcade/issues/6719
+        public static ref T Unbox<T>(object box) where T : struct { throw null; }
```
Comment (dagood) [inline src/referencePackages/src/system.runtime.compilerservices.unsafe/5.0.0/lib/netcoreapp2.0/System.Runtime.CompilerServices.Unsafe.cs:71]: https://github.com/dotnet/arcade/issues/6719

**Comment by dagood** [comment]
Filed https://github.com/dotnet/source-build/issues/1957 for `<icon>Icon.png</icon>` handling.

**Pull request merged**